### ISSUE TITLE
Disable keyboard focus highlight on touch devices

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-grid#277]
The blue underline for a row should be removed on touch devices, as it is only relevant for keyboard navigation.

**Comment by jounik** [comment]
The keyboard focus might be useful on Android devices, but on iOS it’s useless since the arrow keys do nothing expect scroll the whole page.

**Comment by jouni** [comment]
Wontfix. 2.0 does not have this issue (keyboard focus is only shown when using the keyboard).
